craft_providers/bases/buildd.py

```diff
@@ -299,6 +350,19 @@ def _setup_hostname(self, *, executor: Executor, deadline: Optional[float]) -> N
                 details=errors.details_from_called_process_error(error),
             ) from error
 
+    def _setup_instance_config(
+        self, *, executor: Executor, deadline: Optional[float]
+    ) -> None:
+        config = instance_config.InstanceConfiguration(
+            compatibility_tag=self.compatibility_tag
+        )
```

Comment:
this multiline thing does fit in 99c, right?

Reply:
I believe this is just black doing its thing